src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -59,15 +59,14 @@ class ModuleRegistry(private val config: Config) {
 
     private val modules = mutableListOf<Entry>()
 
-    fun getModules(): List<Entry> {
-        val onlyModules = modules
-            .filter { config[it.config.only] }
-        return if (onlyModules.isEmpty()) {
-            modules
-        } else {
-            onlyModules
-        }
-    }
+    fun getAllModules(): List<Entry> = modules
```

Comment:
I really wish we could make this entire class more testable, but out of scope for this PR

Reply:
Yeah :/ I needed to add that method just for the testing. Would have liked to avoid it.